src/Files/MultilingualResources/Files.de-DE.xlf

```diff
@@ -60,7 +60,7 @@
         </trans-unit>
         <trans-unit id="SettingsOnStartupContinueWhereYouLeftOff.Content" translate="yes" xml:space="preserve">
           <source>Continue where you left off</source>
-          <target state="new"></target>
+          <target state="translated"></target>
```

Comment:
@R3voA3 there is a blank string here, is that intended?

Reply:
😱